corehq/messaging/smsbackends/http/models.py

```diff
@@ -73,25 +73,39 @@ def send(self, msg, *args, **kwargs):
         verify_sms_url(config.url, msg, backend=self)
 
         url_params = urlencode(params)
+        ssl_context = self.create_ssl_context()
         try:
-            unverified = ssl._create_unverified_context()
             if config.method == "GET":
                 urlopen(
                     "%s?%s" % (config.url, url_params),
-                    context=unverified,
+                    context=ssl_context,
```

Comment:
It's hard to see where the URL is coming from in this code review interface, but I'm guessing it's different for different SMS gateways. Should this context be applied to every URL, or would it make sense to apply it only in cases where we know it is needed?